packages/design-system-web/src/components/geov-entity-download-rdf/geov-entity-download-rdf.tsx

```diff
@@ -0,0 +1,117 @@
+import { Component, Host, h, Prop } from '@stencil/core';
+import { Color } from '@ionic/core';
+import { State } from '@stencil/core/internal';
+
+@Component({
+  tag: 'geov-entity-download-rdf',
+  styleUrl: 'geov-entity-download-rdf.css',
+  shadow: false,
+})
+export class GeovEntityDownloadRdf {
+  /**
+   * entityId
+   * ID number of entity, e.g. 'i315800'
+   */
+  @Prop() entityId: string;
+  /**
+   * color
+   * color of the button
+   */
+  @Prop() color?: Color;
+  /**
+   * expand
+   * expand of the button
+   */
+  @Prop() expand?: 'block' | 'full';
+  /**
+   * fill
+   * fill of the button
+   */
+  @Prop() fill?: 'clear' | 'outline' | 'solid' | 'default';
+  /**
+   * buttonLabel
+   * Label of the button
+   */
+  @Prop() buttonLabel = 'Download RDF';
+  /**
+   * buttonIcon
+   * Icon of the button
+   */
+  @Prop() buttonIcon = 'download-outline';
+  /**
+   * listFormat
+   * List or RDF serialization format
+   */
+  @State() listFormat: Record<string, string[]> = {
+    'RDF XML': ['application/rdf+xml', '.rdf'],
+    'JSON-LD': ['application/ld+json', '.jsonld'],
+    'N-Triples': ['application/n-triples', '.nt'],
+    'N-Quads': ['application/n-quads', '.nq'],
+    'TRIX': ['application/trix+xml', '.trix'],
+    'Thrift': ['application/rdf+thrift', '.thrift'],
+    'Turtle': ['text/turtle', '.ttl'],
+  };
+  /**
+   * File for download
+   */
+  @State() response: Blob;
+
+  modal: HTMLIonModalElement;
+
+  render() {
+    return (
+      <Host>
+        <ion-button id="open-custom-dialog" expand={this.expand} fill={this.fill} color={this.color} onClick={() => (this.modal.isOpen = true)}>
+          {this.buttonLabel} <ion-icon name={this.buttonIcon}></ion-icon>
```

Comment:
if we use ion-icon attribute "name", the ion-icon will be requested by the browser (e.g. https://foo.bar/svg/download.svg) – thus the server needs to provide the icons.
To create standalone web components we can also inline the svg by using the ion-icon attribute "icon" and pass in the icon.
IMO we should use the second method. To keep the icon customizable, I suggest to use a defoult svg icon, in case no specific icon is set.